mono/metadata/jit-icall-reg.h

```diff
@@ -350,3 +350,15 @@ mono_check_jit_icall_info (gconstpointer jit_icall_info)
 
 	return (MonoJitICallInfo*)jit_icall_info;
 }
+
+static inline MonoJitICallInfo*
+mono_jit_icall_info_at (int index)
+{
```

Comment:
I would call this mono_find_jit_icall_info (). It should also take an enum typed argument.

Reply:
Both done now. We'll see if C++ compiles. :)